content/zh-cn/docs/contribute/style/style-guide.md

```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
 将 `replicas` 字段值设置为 2. | 将 `replicas` 字段值设置为 `2`.
 {{< /table >}}
 
+<!--
+## Referring to Kubernetes API resources
+
+This section talks about how we reference API resources in the documentation.
+
+### Clarification about "resource"
+
+Kubernetes uses the word "resource" to refer to API resources, such as `pod`, `deployment`, and so on. We also use "resource" to talk about CPU and memory requests and limits. Always refer to API resources as "API resources" to avoid confusion with CPU and memory resources.
+-->
+## 引用 Kubernetes API 资源
+
+本节讨论我们如何在文档中引用 API 资源。
+
+### 有关 “资源” 的阐述
+
+Kubernetes 使用 “resource” 一词来指代 API 资源，例如 `pod`、`deployment` 等。 
+我们还使用 “resource” 来谈论 CPU 和内存请求和限制。 
+所以始终将 API 资源称为 “API resources” 以避免与 CPU 和内存资源混淆。
+
+<!--
+### When to use Kubernetes API terminologies
+
+The different Kubernetes API terminologies are:
+
+- Resource type: the name used in the API URL (such as `pods`, `namespaces`)
+- Resource: a single instance of a resource type (such as `pod`, `secret`)
+- Object: a resource that serves as a "record of intent". An object is a desired state for a specific part of your cluster, which the Kubernetes control plane tries to maintain.
+
+Always use "resource" or "object" when referring to an API resource in docs. For example, use "a `Secret` object" over just "a `Secret`".
+-->
+### 何时使用 Kubernetes API 术语
+
+不同 Kubernetes API 术语的说明如下：
+
+- 资源类型：API URL 中使用的名称（如 `pods`、`namespaces`）
+- 资源：资源类型的单个实例（如 `pod`、`secret`）
+- 对象：作为 “意向记录  ” 的资源。对象是集群特定部分的期望状态， 
+  Kubernetes 控制平面试图维护该状态。
+
+在文档中引用 API 资源时始终使用 “资源” 或 “对象”。 
+例如，使用 “一个 `Secret` 对象” 而不是 “一个 `Secret`”。
+
+<!--
+### API resource names
+
+Always format API resource names using [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as PascalCase, and code formatting.
+
+For inline code in an HTML document, use the `<code>` tag. In a Markdown document, use the backtick (`` ` ``).
+
+Don't split an API object name into separate words. For example, use `PodTemplateList`, not Pod Template List.
+
+For more information about PascalCase and code formatting, please review the related guidance on [Use upper camel case for API objects](/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) and [Use code style for inline code, commands, and API objects](/docs/contribute/style/style-guide/#code-style-inline-code).
+
+For more information about Kubernetes API terminologies, please review the related guidance on [Kubernetes API terminology](/docs/reference/using-api/api-concepts/#standard-api-terminology).
+-->
+### API 资源名称
+
+始终使用 [UpperCamelCase](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB) 
+（也称为 PascalCase）和代码格式来表达 API 资源名称。
```

Comment:
The English word "UpperCamelCase" is not a proper noun, just "upper camel case" written in pascal case.

```suggestion
始终使用[大寫駝峰式命名法](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB)
（PascalCase）和代码格式来表达 API 资源名称。
```